README.md

```diff
@@ -29,3 +30,29 @@ You can use a different registry by overwriting the `IMAGE_REGISTRY` variable:
 ```shell
 make -e IMAGE_REGISRY=quay.io/YOUR_USER/notebooks base-ubi8-python-3.8
 ```
+
+### Building jupyter-minimal-ubi8-python-3.8 image
+
+The jupyter-minimal-ubi8-python-3.8 is a lightweight image for running the Jupyter notebook server.
+
+We use the base-ubi8-python-3.8 image [quay.io/opendatahub/notebooks:base-ubi8-python-3.8](https://quay.io/repository/opendatahub/notebooks?tab=tags) as a builder image.
+
+Build the `jupyter-minimal-ubi8-python-3.8` by running the following command:
+
+```shell
+make jupyter-minimal-ubi8-python-3.8
+```
+
+The image will be built and pushed to the [quay.io/opendatahub/notebooks](https://quay.io/opendatahub/notebooks) repository.
+
+You can use a different registry by overwriting the `IMAGE_REGISTRY` variable:
+
+```shell
+make -e IMAGE_REGISRY=quay.io/YOUR_USER/notebooks jupyter-minimal-ubi8-python-3.8
+```
+
+The following command starts a container with the Notebook server listening for HTTP connections on port 8888 with a randomly generated authentication token configured.
+
+```shell
+podman run -it --rm -p 8888:8888 quay.io/opendatahub/notebooks:jupyter-minimal-ubi8-python-3.8
+```
```

Comment:
This is a debug command, shouldn't be included here. We'll create some instructions for running the notebooks in Kubernetes/OpenShift when we have the corresponding manifests ready.